### PR TITLE
[Imager] Add common checks folder and fix nits in HACKING.md

### PR DESCRIPTION
**Pull request by leo8a**
/cc @tsorya @omertuc

### CODE REVIEW

File: ibu-imager/common/consts.go [hunk 1]
```diff
@@ -18,13 +18,35 @@ package common
...
+	EtcdStaticPodContainer = "etcd"
+
 	InstallationConfigurationFilesDir = "/usr/local/installation_configuration_files"
 )
+
+// FilesToRemove is the list of systemd service units, scripts, and env file
+// to be cleaned up after creating the seed image
+var FilesToRemove = []string{
+	"/etc/systemd/system/installation-configuration.env",
+	"/var/usrlocal/bin/installation-configuration.sh",
```
Comment (tsorya) [inline ibu-imager/common/consts.go:41]: Can we read files from the folder and remove accordingly ? I would like not to use list of files if it is possible

File: ibu-imager/common/consts.go [hunk 2]
```diff
@@ -18,13 +18,35 @@ package common
...
+	"/var/usrlocal/bin/prepare-installation-configuration.sh",
+	"/etc/systemd/system/installation-configuration.service",
+	"/etc/systemd/system/prepare-installation-configuration.service",
+}
+
+// SystemdUnitsToDisable is the list of systemd units to be disabled
+// after creating the seed image
+var SystemdUnitsToDisable = []string{
+	"installation-configuration.service",
```
Comment (tsorya) [inline ibu-imager/common/consts.go:50]: same

File: ibu-imager/common/consts.go (hunk 1 above)
Comment (tsorya) [reply ibu-imager/common/consts.go:41]: Sort of same as https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/cmd/create.go#L133 
just need to take file name and build path accordingly

Comment (leo8a) [reply ibu-imager/common/consts.go]: agree, will do it as suggested in #79 :)

**Review by donpenney — APPROVED** [review]

**Pull request merged**